Tools-Override/tests.targets

```diff
@@ -25,10 +25,16 @@
     <UnsupportedPlatformsItems Include="$(UnsupportedPlatforms)"/>
   </ItemGroup>
 
+  <!-- This is required to ensure that every test project has a copy of the xunit runner app -->
+  <ItemGroup>
+    <SupplementalTestData Include="$(RuntimePath)\xunit.console.netcore.exe" />
```

Comment:
Can you put this in the same location where we are doing the runtimeconfig below in this targets file?